UnicornEngine/video/source/vulkan/Renderer.cpp

```diff
@@ -1175,11 +1334,31 @@ bool Renderer::CreateCommandBuffers()
             {
                 if(pVkMesh->IsValid())
                 {
+                    glm::vec4 colorPush({pVkMesh->GetColor(), pVkMesh->IsColored()}); // xyz - color. w - 1.0 enabled color or 0.0 disabled color
```

Comment:
Comment doesn't make much sense. What is `w`?

Reply:
Boolean flag. Changed to 
 // xyz - color. w - boolean flag for 1 enabled color or 0 disabled color